README.md

```diff
@@ -48,21 +181,33 @@ The following is a non-normative example of a W3C VC using the external [JWT pro
     ],
     "type": [
       "VerifiableCredential",
-      "UniversityDegreeCredential"
+      "IDCardredential"
     ],
     "credentialSubject": {
-      "degree": {
-        "type": "BachelorDegree",
-        "name": "Bachelor of Computer Science"
-      }
+      "given_name": "Fredrik",
+      "family_name": "Strömberg",
+      "birthdate": "1949-01-22",
+      "place_of_birth": {
+        "country": "SE",
+        "locality": "Örnsköldsvik"
+      },
+      "nationality": "SE",
+      "number": "4901224131",
+      "date_of_issuance":"2010-03-23",
+      "date_of_expiry":"2020-03-22"
+      "issuer":{
```

Comment:
@tlodderstedt the `issuer` property should be one level above.

Reply:
just as a claim within the vc container?

Reply:
done